src/Groups/types.jl

```diff
@@ -238,13 +238,22 @@ Group of automorphisms over a group of type `T`. It can be defined via the funct
 mutable struct AutomorphismGroup{T} <: GAPGroup
   X::GapObj
   G::T
+  to_gap
+  to_oscar
```

Comment:
Let's not put them into all instances, instead, use `set_special(autG, :to_gap => BLA)` and `get_special(autG, :to_gap)` to set/get them (resp. very soon: `set_attribute` and `get_attribute`, see https://github.com/oscar-system/Oscar.jl/issues/843 and https://github.com/Nemocas/AbstractAlgebra.jl/pull/1065).